resources/views/pages/donation-response.blade.php

```diff
@@ -0,0 +1,15 @@
+@extends('master')
+
+@section('title')
+	Donatie
+@endsection
+
+@section('content')
+
+<h1>Donatie</h1>
+
+<hr/>
+
+<p>Je donatie is ontvangen! Bedankt!</p>
```

Comment:
Weet niet helemaal waarom ik de code suggestion knop niet meer zie, maar...

Gezien de taal op de site ook formeel is, zou ik hier "Uw donatie" van maken.

Reply:
done